components/tidb_query/src/rpn_expr/impl_string.rs

```diff
@@ -409,6 +409,15 @@ pub fn find_in_set(s: &Option<Bytes>, str_list: &Option<Bytes>) -> Result<Option
     })
 }
 
+#[rpn_fn]
+#[inline]
+pub fn trim_1_arg(arg: &Option<Bytes>) -> Result<Option<Bytes>> {
+    Ok(arg.as_ref().map(|bytes| {
+        let s = String::from_utf8_lossy(bytes);
```

Comment:
Here we don't need to allocate a new string. Using `std::str::from_utf8` can save an allocation.

Reply:
Done.